server/utils.go

```diff
@@ -83,3 +83,15 @@ func closeBody(r *http.Response) {
 		r.Body.Close()
 	}
 }
+
+func isValidAuthConfig(configuration *configuration) (bool, error) {
+	switch {
+	case configuration.EnableLegacyAuth && configuration.EnableOAuth:
+		return false, errors.New("Only enable One of the OAuth or Password based authentication")
```

Comment:
```suggestion
		return false, errors.New("Only one authentication scheme (OAuth or Password) is allowed to be enabled at the same time.")
```

Reply:
Thanks for the suggestion @cpoile, updated the message